Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 
 [dependencies]
 proj-sys = { version = "0.18.3", path = "proj-sys" }
-geo-types ="0.6.0"
+geo-types = { version = "0.6", optional = true }
```

Comment:
I snuck this in - there wasn't a reason for us to peg the patch version of geo-types was there?